### PR TITLE
#175756608 Created a Home Page

### PR DESCRIPTION
**Pull request by dianahub250**
#### What does this PR do?
Creating a home page for Visit Africa App
#### Description of Task to be completed?
- The user can be able to view most visited places
-The use must be able to search 
-The user can be able to navigate to side menu
#### How should this be manually tested?
-Clone it
-Run flutter pub get to install dependencies
-Run it by using flutter run command
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#https://www.pivotaltracker.com/story/show/175756608
#### Screenshots (if appropriate)
Screenshot_1606125218

#### Questions:
N/A

### CODE REVIEW

**Review by Rita-12397 — APPROVED** [review]

**Review by Sandrine21 — APPROVED** [review]
Good, you can conyinue.

File: lib/main.dart [hunk 1]
```diff
@@ -1,117 +1,10 @@
...
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
-    );
-  }
+  runApp(MaterialApp(
+    title: 'Todos ',
```
Comment (mujohn26) [inline lib/main.dart]: What's the meaning of this tittle ?

File: test/widget_test.dart [hunk 2]
```diff
@@ -1,30 +1,30 @@
-// This is a basic Flutter widget test.
-//
-// To perform an interaction with a widget in your test, use the WidgetTester
-// utility that Flutter provides. For example, you can send tap and scroll
-// gestures. You can also use WidgetTester to find child widgets in the widget
-// tree, read text, and verify that the values of widget properties are correct.
+// // This is a basic Flutter widget test.
```
Comment (mujohn26) [inline test/widget_test.dart:1]: What is the use of this file ? I see it's a commented line.

**Review by mujohn26 — CHANGES_REQUESTED** [review]

**Review by mujohn26 — APPROVED** [review]
Good

**Pull request merged**